README.md

```diff
@@ -107,4 +107,9 @@ Now that Vale is installed you can check individual files or directories locally
 vale docs
 vale docs/*.md
 vale docs/test.md
-```
\ No newline at end of file
+```
+
+### Customising the rules
+
+The published Vale Github action ignores suggestions and turns off some features which are likely to cause a lot of noise if run in their default state. A lot of things boil down to spellings. For example, the rule which checks for sentence case in headings doesn't know that SATA is an initialism by default, so if you include it, completely reasonably, in a heading it will be marked as an error. You may wish to turn this back on **if** you have a fairly complete vocabluary list.
```

Comment:
small typo: vocabluary => vocabulary.

Reply:
should the vale action be enabled on this repo? 😄